homework/shelkovich_ilya/userValidationService/src/test/java/com/tutrit/java/quickstart/service/UserValidationServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.User;
+import com.tutrit.java.quickstart.exceptions.UserValidationException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UserValidationServiceTest {
+
+    UserValidationService userValidationService;
+    User user;
+
+    @Before
+    public void setUp() {
+        this.userValidationService = new UserValidationService();
+        this.user = new User();
```

Comment:
I guess keyword "this" is redundant here. 
You are referring to the field of this class.

Reply:
I prefer to evidently point that var belong to this object. Using this is normal.

Reply:
fixed